rocks-lib/src/tree/mod.rs

```diff
@@ -50,6 +50,10 @@ impl Tree {
         self.root.join(self.version.to_string())
     }
 
+    pub fn root_for(&self, rock_name: &PackageName, rock_version: &PackageVersion) -> PathBuf {
```

Comment:
suggestion: why not take a `&LuaPackage` as an argument?

Reply:
That's to keep consistency with the `rock()` function, which also takes a name and version. Those should be refactored to just use `LuaPackage` eventually though.